src/defaultPropTypes.js

```diff
@@ -17,11 +17,11 @@ export default {
   activeBarStyle: PropTypes.object,
   dismissAfter: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.number
+    PropTypes.number,
```

Comment:
No, we should not have dangling commas. Thanks for catching this, @BerkeleyTrue.